db/migrate/20180513172545_create_customers.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCustomers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :customers do |t|
+      t.string :name
```

Comment:
It's usually a good idea to add validations to your database as well as to your model. So for ex.:
`t.string :name, :null => false`
Read more: https://stackoverflow.com/questions/13122791/rails-validation-in-model-vs-migration?noredirect=1&lq=1